pkg/reconciler/kubernetes/tektonconfig/tektonconfig.go

```diff
@@ -125,6 +125,10 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tc *v1alpha1.TektonConfi
 		}
 	}
 
+	if err := common.Prune(r.kubeClientSet, ctx, tc); err != nil {
+		logger.Error(err)
+	}
+
```

Comment:
This statement here breaks the flow of logic here. This stands out between `stages` and `stages execute` .
Move this to postReconcile. https://github.com/tektoncd/operator/blob/05659f93604d4741977d47e72c58dca4f609fed6/pkg/reconciler/kubernetes/tektonconfig/extension.go#L51